ci/provisioning/post_provision_config_nodes_LEAP_15.sh

```diff
@@ -67,4 +67,8 @@ post_provision_config_nodes() {
         done
         exit "$rc"
     fi
-}
+
+    # now make sure everything is fully up-to-date
+    zypper addlock daos daos-\*
```

Comment:
Should there be locks on fuse and mercury, similar to how they're excluded on centos?

Reply:
`zypper` seems to figure out that they shouldn't be upgraded.  I think DNF is that smart too.  I think it's just ancient YUM that has not been updated to more turn-of-the-century resolving logic.